### DISCUSSION TITLE
Remote database relationship: make request and filter with remote field

### DISCUSSION THREAD

**Discussion by GilYounicorns** [hasura/graphql-engine#8464] [General]
Hello, i created hasura project with two databases and i created remote database relationship between them. In my database "Admin" i have table users. i created in this table remote object relation named company with table companies in database "InfoGreffe". 
Now i want to get all users who has a company in 2020 but i got this error :  Uncaught (in promise) Error: field "company" not found in type: 'users'.Here is my request
query: gql `query {
           users(where: {
              company: { created: { _eq: 2020 } },
            }) {
              id
              username
              company {
                id
                name
              }
            }
          }
        `
I can't identify what is wrong with my request. Can you help me?

**Comment by seanonthenet** [comment]
Hi. Have you created a relationship between the tables too? 
https://hasura.io/docs/latest/graphql/core/databases/postgres/schema/table-relationships/create/#introduction
